en/kubernetes-service/k8s-create-connection.md

```diff
@@ -2,3 +2,17 @@
 title: "Kubernetes: Create Service Connection"
 slug: k8s-create-connection
 ---
+
+
+### Create service connection
+
+1. Select connection type **kubernetes-standalone**.
```

Comment:
Should we be incrementing the numbers? Same question for the "working with instances" doc.

Reply:
The "1." is markdown notation, when the doc gets rendered the numbers will increment properly :)